objective.py

```diff
@@ -6,54 +6,127 @@
 with safe_import_context() as import_ctx:
     from sklearn.dummy import DummyClassifier
     from sklearn.model_selection import train_test_split
-
+    from sklearn.metrics import balanced_accuracy_score as BAS
+    import numpy as np
+    from skorch.helper import SliceDataset, to_numpy
 
 # The benchmark objective must be named `Objective` and
 # inherit from `BaseObjective` for `benchopt` to work properly.
+
+
 class Objective(BaseObjective):
 
     # Name to select the objective in the CLI and to display the results.
     name = "BCI"
 
-    requirements = ["scikit-learn"]
-
     # List of parameters for the objective. The benchmark will consider
     # the cross product for each key in the dictionary.
     # All parameters 'p' defined here are available as 'self.p'.
-    # This means the OLS objective will have a parameter `self.whiten_y`.
+
+    link = 'pip: git+https://github.com/Neurotechx/moabb@develop#egg=moabb'
+    intall_cmd = 'conda'
+    requirements = [str(link),
+                    'scikit-learn']
+
     parameters = {
-        'seed': [42],
+        'evaluation_process, subject, subject_test, session_test': [
+            ('intra_subject', 1, None, None),
+             ],
```

Comment:
align this closing bracket with the beginning of the line where the matching `[` is.